src/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="shortcut icon" href="#">
+  <title>Todo List</title>
+</head>
+<body>
+  <div class="todo-list">
+    <div class="task-outer-container">
+      <div class="title-container">
+        <h1 class="tasks-container-title">Today's To Do</h1>
+      </div>
+      <div class="form-container">
+        <input type="text" name="taskForm" id="taskForm" class="task-form" placeholder="Add to your list...">
+      </div>
```

Comment:
I would recommend changing the <div> 'form-container' to an actual <form> because it would allow the user to add a task and hit 'Enter' on the keyboard, making it easier to add tasks.

Reply:
Could not make this specific change of changing the `<div>` to a `<form>`, because it reloads the page each time it sends data, so I used a different approach that has the same effect as your review. Check it in the `event listener` review.

Reply:
I believe the 'reload problem' could be addressed by adding an `e.preventDefault()` on the event listener and calling your function `setTasksLocalStorage(tasksContainer)` and `createTaskHTML()`.
But that's just an alternate option, your implementation works just fine. 🤟